resources/g2/sprites.json

```diff
@@ -197,6 +197,9 @@
     {
         "path": "icons/rct1_simulate_on_pressed.png"
     },
+	{
+		"path": "icons/infinity.png"
+	},
```

Comment:
Use spaces
```suggestion
    {
        "path": "icons/infinity.png"
    },
```

Reply:
Thisone could probably also use `"palette" : "keep"`. That would be ever so slightly faster as the game won't need to convert (as far as I've been told).

Reply:
that seems to work so I'll keep it in